tekton-pipelines/wordpress/build-pipeline.yaml

```diff
@@ -75,4 +78,6 @@ spec:
           value: "$(tasks.wordpress-deployment.status)"
         # - name: pipelinerun_name
         #   value: "$(tasks.get-pipelinerun-name.results.pipelinerun_name)"
-
+      workspaces:
```

Comment:
no need to define workspace for slack notification task if emitting results works
https://tekton.dev/docs/pipelines/tasks/#emitting-results